src/app/js/public/ExportShareButton.js

```diff
@@ -0,0 +1,27 @@
+import React from 'react';
+import { connect } from 'react-redux';
+import translate from 'redux-polyglot/translate';
+import compose from 'recompose/compose';
+
+import { fromExport } from './selectors';
+import ButtonWithDialog from '../lib/components/ButtonWithDialog';
+import { openExport, closeExport } from './export';
+import ExportShare from './ExportShare';
+
+const mapStateToProps = (state, { p, uri, style }) => ({
+    show: true,
```

Comment:
I don't understand why this is not in `defaultProps` rather than `mapStateToProps`